doc/source/projects.rst

```diff
@@ -0,0 +1,74 @@
+Ray Projects (Experimental)
+===========================
+
+Ray projects make it easy to package a Ray application so it can be
+rerun later in the same environment. They allow sharing and reuse
+existing code reliably.
```

Comment:
They allow for the sharing and reuse of existing code reliably.

Reply:
fixed